### DISCUSSION TITLE
java error: Split cannot be done on an overlay block

### DISCUSSION THREAD

**Discussion by nt4boy** [NationalSecurityAgency/ghidra#5253] [Q&A]
I've attempted to import a 16 bit MS DOS application, and this is as far as I can get.
![image](https://user-images.githubusercontent.com/113193761/233842171-fc618f29-8445-49d8-b1d9-c3bc930b140b.png)

I must be doing something stupid, could someone advise please?

Thanks

**Comment by ryanmkurtz** [comment]
Can you attach your binary?

**Comment by nt4boy** [comment]
Ryan,
Thanks for your assistance
The exe file is the one that won’t import.
(If you try and run it in DOS, it requires the EGA library.)

Rupert

**Reply by ryanmkurtz** [reply]
I am not seeing it attached to this discussion.

**Comment by nt4boy** [comment]
Sorry, I don’t think I can make it that public.
Let’s close the issue.
Regards

**Comment by nt4boy** [comment]
I did find a solution to this, but no idea why it works.
In W10 properties, the file is stated as having:-
 Size 111,888 bytes and Size on Disk 114,688 bytes. and errors as above.
I edited the file in FRHED, to change two byte values from 75 to 74 and saved it.
Size is now 114,448 and Size on Disk as before 114,688. This Imports correctly.

**Comment by nt4boy** [comment]
As I say, I don't know how the file stats get created or even if Ghidra uses them, but the error has gone away.
